app/static/main.js

```diff
@@ -2,190 +2,238 @@ let proxIdsSelected = new Set();
+          e.setAttribute("disabled", true);
+        });
+      }
     }
+    return 1;
+  }
 }
 
 function controlInputWeight(event) {
-    const [, type, ind] = event.target.id.toLowerCase().split("_");
-    const value = event.target.value;
-    if (isNaN(value)) {
-        event.target.value = inputWeightState[type][ind];
-    } else if (parseInt(value) >= 1) {
-        inputWeightState[type][ind] = value;
-    } else if (value === "") {
-        inputWeightState[type][ind] = value;
-    } else {
-        event.target.value = inputWeightState[type][ind];
-    }
+  const [, type, ind] = event.target.id.toLowerCase().split("_");
+  const value = event.target.value;
+  if (isNaN(value)) {
+    event.target.value = inputWeightState[type][ind];
+  } else if (parseInt(value) >= 1) {
+    inputWeightState[type][ind] = value;
+  } else if (value === "") {
+    inputWeightState[type][ind] = value;
+  } else {
+    event.target.value = inputWeightState[type][ind];
+  }
 }
 
 function getInputTypeIgnoreCase(inputType) {
-    return inputType.toLowerCase() === TIME ? TIME : CLUSTER;
+  return inputType.toLowerCase() === TIME ? TIME : CLUSTER;
 }
 
 function controlInput(event) {
-    const type = getInputTypeIgnoreCase(event.target.getAttribute("inputType"));
-    const min = event.target.getAttribute("min");
-    const max = event.target.getAttribute("max");
-    const value = event.target.value;
-    if (isNaN(value)) {
-        event.target.value = inputState[type];
-    } else if (parseInt(value) >= min && parseInt(value) <= max) {
-        inputState[type] = value;
-    } else if (value === "") {
-        inputState[type] = value;
-    } else {
-        event.target.value = inputState[type];
-    }
+  const type = getInputTypeIgnoreCase(event.target.getAttribute("inputType"));
+  const min = event.target.getAttribute("min");
+  const max = event.target.getAttribute("max");
+  const value = event.target.value;
+  if (isNaN(value)) {
+    event.target.value = inputState[type];
+  } else if (parseInt(value) >= min && parseInt(value) <= max) {
+    inputState[type] = value;
+  } else if (value === "") {
+    inputState[type] = value;
+  } else {
+    event.target.value = inputState[type];
+  }
+}
+
+function controlDynamicInputs(event) {
+  
+}
+
+function toggleAlgoDropdown(ddButton) {
+  document.querySelector(".algo-dropdown-options").classList.toggle("show");
 }
 
+function handleAlgoDropdownSelect(event) {}
 
 function process() {
+  /*let chart = document.querySelector("#chart");
+  chart.style.maxHeight = document.querySelector(".data-input-container").clientHeight;
+  chart.src = `data:image/png;base64,${imgPath}`;
+  let referenceImg = new Image();
+  referenceImg.onload = () => {
+    chart.style.maxWidth = document.querySelector(".data-input-container").clientHeight * referenceImg.width / referenceImg.height;
+  }
+  referenceImg.src = `data:image/png;base64,${imgPath}`;*/
   /* Request the data and chart */
```

Comment:
Reason for comment vs deleting?

Reply:
I use this when I don't want to wait 5 minutes for process to finish. To test the UI.

Reply:
This kind of code shouldn't be committed.